GemmaM0_Headband/code.py

```diff
@@ -0,0 +1,33 @@
+# Gemma M0 Onchip Temperature Sensor
+# Project by Kathy Ceceri
+# CircuitPython by Anne Barela
+# Adafruit Industries, 2019
+
+import time
+import board
+import microcontroller
+import neopixel
+import adafruit_dotstar
+
+# Set up NeoPixel strand
+pixels = neopixel.NeoPixel(board.D1,  # NeoPixels on pin D1
+                           4,         # Number of Pixels
+                           brightness=0.2,   # Change from 0.0 to 1.0
+                           auto_write=True)  # Write strand immediately
+
+# For the Gemma M0 onboard DotStar LED
+dotstar = adafruit_dotstar.DotStar(board.APA102_SCK, board.APA102_MOSI, 1)
+
+def deg_f(deg_c):  # Convert Celcius to Fahrenheit
+    return(deg_c * 9 / 5) + 32.0
+
+while True:
+    temp = deg_f(microcontroller.cpu.temperature)
+    if temp > 65.0:
```

Comment:
I prefer checking temp against a `CONST` like `ROOM_TEMP` defined at the top of the file

You'd be able to modify this paragraph in the guide to explain it if you want to go this optional route.
> If you need to make any adjustments, such as changing the "room temperature" cut-off that triggers the colors, or you have a different number of pixels attached to your Gemma M0, you can type them right into the code.

Reply:
changed